### ISSUE TITLE
Posting to Friendica (support forum) from Mastodon doesn't work?

### ISSUE THREAD

**Issue by smth** [friendica/friendica#4979]
### Expected behavior
See the post on https://forum.friendi.ca/~helpers

### Actual behavior
Don't see the post on https://forum.friendi.ca/~helpers

### Steps to reproduce the problem
1. Create a Mastodon account (I did this on mstdn.io)
2. Connect with helpers
3. Mention [helpers](https://forum.friendi.ca/profile/helpers) in a "[toot](https://mstdn.io/@smth/99968068433985555)" from Mastodon

(This is my first time posting on Mastodon, or to Friendica, so entirely possible I'm just doing it wrong. Apologies in advance.)

**Comment by annando** [comment]
Yes, this currently doesn't work. The group support from other networks than Friendica is very limited.
